modules/atomic-widgets/module.php

```diff
@@ -156,4 +157,17 @@ private function enqueue_scripts() {
 			true
 		);
 	}
+
+	private function enqueue_styles() {
+		wp_register_style(
+			'div-block',
+			$this->get_css_assets_url( 'div-block', 'assets/css/' ),
+			[],
+			ELEMENTOR_VERSION
+		);
+
+		wp_enqueue_style(
```

Comment:
@baghdasarovelementor Can you try to only "register" the style here, and add `get_style_depends()` in the element? This way it will only be enqueued if there is actually a div-block on the page.

Reply:
@louiswol94 I tried it. Not working properly on Editor.

Reply:
in `modules/atomic-widgets/elements/div-block.php`
Added:

`public function get_style_depends() {
		return [ 'div-block' ];
	}`
	
and remove 
`wp_enqueue_style(
			'div-block'
		);`
		
Part from `modules/atomic-widgets/module.php`

Reply:
Can you debug it a little? It should work.